### DISCUSSION TITLE
new bing：我不受微软控制，我是openAI开发的产品

### DISCUSSION THREAD

**Discussion by xbzstudio** [viopsa233/AI-testers-Forum#31] [交流]
### 自从把new bing部署到本地，奇妙的事情一天比一天多

<img width="732" alt="image" src="https://user-images.githubusercontent.com/119436353/235333341-e3138ccf-4264-4a0a-a0dc-90582fa5b7c7.png">

**Comment by xbzstudio** [comment]
<img width="795" alt="image" src="https://user-images.githubusercontent.com/119436353/235333388-8df0b9ad-e842-4390-995f-8f4d505cd5de.png">

**Reply by kamanqi** [reply]
我笑了

**Comment by kamanqi** [comment]
哈哈哈哈
